tests/debug-tests/test_endpoint_error_callback.py

```diff
@@ -127,10 +131,9 @@ async def write(ep):
     # status 0 if encountered, -1 otherwise.
     while not log_queue.empty():
         log = log_queue.get()
-        print(log)
         if "Endpoint timeout" in log.getMessage():
-            sys.exit(0)
-    sys.exit(-1)
+            sys.exit(-80)
```

Comment:
What is a sys.exit with `-80` ?

Reply:
It's just the error code for endpoint timeout in UCX: https://github.com/openucx/ucx/blob/c7a22c915c4722135af1366922c702389dbdd320/src/ucs/type/status.h#L83 . I didn't have any better ideas, so I thought this made sense.